http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -111,16 +129,79 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
-}
+  private def setUpMockIdentity: Identity = {
```

Comment:
Updated these tests to use Mockito -- it would be hard to mock out Azure SDKs otherwise